userdocs/src/components/kubernetes.md

```diff
@@ -48,6 +49,29 @@ data:
   admin_password: ...
 ```
 
+Inspect the secrets you just applied with:
+
+```bash
+kubectl get secrets --namespace {your-namespace}
+kubectl describe secret postgres-config --namespace {your-namespace}
+```
+
+If you have a postgres `Deployment`, add the following `env` configuration to let your pods get the `POSTGRES_PASSWORD`
```

Comment:
Should we explicitly state that this deployment needs to be applied within the same namespace as the secret above?

Reply:
Good idea, fixed.